app/wayfinders/base_wayfinder.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+class BaseWayfinder
```

Comment:
Can you add class level docs about the intended use of this class? e.g. use in decorators. Also we had talked about using in controllers. It seems like that would still work; maybe a little something about what that would look like would fit here.